scripts/player.gd

```diff
@@ -75,6 +85,9 @@ func _physics_process(delta):
 
 	if not is_on_floor():
 		velocity.y += gravity * delta
+		# only airbrake if the player is not otherwise moving
```

Comment:
I feel like this should maybe halt upwards velocity as well. Makes sense that it wouldn't affect downward due to gravity.